decidim-proposals/lib/decidim/proposals/markdown2proposals.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+require "redcarpet"
+
+module Decidim
+  module Proposals
+    # This class parses a participatory text document in markdown and
+    # produces Proposalsin the form of sections and articles
+    class Markdown2Proposals < ::Redcarpet::Render::Base
```

Comment:
Please don't use abbreviations: `MarkdownToProposals`

Reply:
Sorry, I'm going to leave it as `Markdown2Proposals` as the use of xxx2yyy is a well known terminology in software.

Reply:
It's a long discussion with you @tramuntanal. Please, as we will have to maintain the software, do not use abbreviations.

Thanks.

Reply:
C'mon, are you joking? This is not an abbreviation

Reply:
It's an obvious abbreviation: you're shortening "to" to "2", thus saving 1 single character. It even appears in some dictionaries as an abbreviation:

https://www.macmillandictionary.com/dictionary/british/2

```suggestion
    class MarkdownToProposals < ::Redcarpet::Render::Base
```